### DISCUSSION TITLE
New Releases

### DISCUSSION THREAD

**Discussion by Bofundu** [SpotX-Official/SpotX#28] [Ideas]
Would it be possible to add 'new releases' section somewhere? I think Spicetify has that feature.

**Comment by amd64fox** [comment]
You can use `Spicetify` in combination with `SpotХ`.

For example:

1. First install SpotX, then install Spicetify on top and customize it to your taste.

Or you can do the opposite.

2. first install Spicetify, customize it to your taste, and then install SpotХ on top.

But if you use the second case, then when you try to use the Spicetify commands, my patch will be reset, you will need to install it on top again.
